src/pocketmine/plugin/PluginDescription.php

```diff
@@ -45,6 +45,17 @@ class PluginDescription{
 	private $permissions = [];
 
 	/**
+	 * @param string|array $str
+	 */
+	static protected function __toArray($str){
```

Comment:
Yes, didn't know what to name the function.  Can anybody suggest a better name?
It is meant to make sure that arrays are arrays (or that strings are converted to arrays).